app/Resources/views/deletionCounter/result.html.twig

```diff
@@ -0,0 +1,62 @@
+{% extends 'base.html.twig' %}
+{% import 'macros/layout.html.twig' as layout %}
+{% import 'macros/wiki.html.twig' as wiki %}
+{% import 'macros/pieChart.html.twig' as chart %}
+
+{% block body %}
+<div class="panel panel-primary">
+    {{ layout.userHeading(user, project, xtPage) }}
+
+    <div class="panel-body xt-panel-body">
+        {{ wiki.userLinks(user, project, xtPage) }}
+
+        {% set content %}
+            <div class="col-lg-6">
+            <table class="table table-bordered table-hover table-striped xt-show-hide--target">
+                <thead>
+                <tr>
+                    <th>{{ msg("deletion-process") }}</th>
```

Comment:
Before we commit this into version control and the translators get to work, we might consider just using the term "type" for now. I say this because eventually it'd be nice if we had a breakdown of the types of deletions for a given process (A1, A2, etc.). For now "type" is still a fitting word for this column and we already have a message for it.